src/Features/Core/Portable/Shared/Extensions/ISymbolExtensions_2.cs

```diff
@@ -193,6 +201,13 @@ private static string GetDocumentation(ISymbol symbol, Compilation compilation,
                 _ => symbol.GetDocumentationComment(compilation, expandIncludes: true, expandInheritdoc: true, cancellationToken: cancellationToken).SummaryText,
             };
 
+        private static string GetReturnsDocumentation(ISymbol symbol, Compilation compilation, CancellationToken cancellationToken)
```

Comment:
Does this have to do anything funky with delegates?

Reply:
I'll add tests